### PR TITLE
fix readme link to minikube examples

### PR DESCRIPTION
**Pull request by prasadkatti**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,9 +49,9 @@ See the [Getting Started Guide](https://minikube.sigs.k8s.io/docs/start/)
 
 See https://minikube.sigs.k8s.io/docs/
 
-## More Examples
+## Minikube in action
```
Comment (medyagh) [inline README.md:52]: minikube should always be lowercase ! we are mini :)

Comment (medyagh) [reply README.md:52]: lets keep it as More Examples , I like that one better

File: README.md [hunk 2]
```diff
@@ -49,9 +49,9 @@ See the [Getting Started Guide](https://minikube.sigs.k8s.io/docs/start/)
...
 
-See our [examples page](https://minikube.sigs.k8s.io/docs/examples/)
+See minikube in action [here](https://minikube.sigs.k8s.io/docs/handbook/controls/)
```
Comment (medyagh) [inline README.md:54]: lets add an alias 
so anyone linked to this would be reffered to old location

Comment (medyagh) [reply README.md:54]: here is example of alias https://github.com/kubernetes/minikube/pull/7485

Comment (prasadkatti) [reply README.md:54]: do you mean people with the old link would be referred to the new page?

Comment (prasadkatti) [reply README.md:54]: @medyagh, I have added the alias and confirmed that it works using the netlify link.

**Comment by prasadkatti** [comment]
@medyagh - This link (https://minikube.sigs.k8s.io/docs/reference/networking/) is broken but I can't find that page on the new site.

**Pull request merged**